i18n/en.json

```diff
@@ -3899,6 +3897,11 @@
   "navbar_dropdown.teamSettings": "Team Settings",
   "navbar_dropdown.tryTrialNow": "Try Enterprise for free now!",
   "navbar_dropdown.userGroups": "User Groups",
+  "navbar_dropdown.userGroups.modal.description": "Create unlimited user groups with one of our paid plans. Get the full experience of Enterprise when you start a free, 30 day trial.",
+  "navbar_dropdown.userGroups.modal.description.afterTrial": "User groups are a way to organize users and apply actions to all users within that group. Upgrade to the Professional plan to create unlimited user groups.",
+  "navbar_dropdown.userGroups.modal.title": "Try unlimited user groups with a free trial",
+  "navbar_dropdown.userGroups.modal.title.afterTrial": "Upgrade to create unlimited user groups",
+  "navbar_dropdown.userGroups.tooltip.cloudFreeTrial": "During your trial you are able to create user groups. These user groups will be archived after your trial.",
```

Comment:
@annelieseklein  Used these `navbar_dropdown.userGroups` messages above to speed up development. Please let us know if any change is required.
cc @marianunez

Reply:
Looks great! Thanks so much @julmondragon